ui/apps/platform/cypress/integration/networkGraph/networkGraph.test.js

```diff
@@ -168,6 +168,8 @@ describe('Network Graph smoke tests', () => {
 
         // Check that the values are removed
         cy.get(networkGraphSelectors.manageCidrBlocksButton).click();
-        cy.get(networkGraphSelectors.cidrBlockEntryNameInputAt(0)).should('not.exist');
```

Comment:
This assertion almost always succeeds, because the CIDR entry inputs are only displayed conditionally _after_ the completion of a network request that is initiated when the modal is opened.

Recent CI jobs have started failing, presumably because the assertion is winning the race condition against the network request https://prow.ci.openshift.org/job-history/gs/test-platform-results/logs/branch-ci-stackrox-stackrox-release-4.6-merge-gke-ui-e2e-tests. It Isn't clear why this is happening in this environment.

Regardless, the following change more accurately tests the desired behavior here - that re-opening a modal with all CIDR blocks deleted correctly shows a single set of empty input elements.